.github/dependabot.yml

```diff
@@ -25,6 +29,9 @@ updates:
       - dependency-name: "*"
         # Only create PRs for new patch releases
         update-types: [ "version-update:semver-major", "version-update:semver-minor" ]
+      - dependency-name: "com.amazonaws:aws-java-sdk-core"
```

Comment:
We can still receive dependabot PRs for this dep, but at a less frequent rate by  adding an additional config like:

```yaml
  - package-ecosystem: maven
    directory: "/"
    target-branch: "14.0.x"
    schedule:
      interval: "weekly"
    allow:
      - dependency-name: "com.amazonaws:*"
```